gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.0.1-bin.zip
```

Comment:
Are you sure you want to upgrade Gradle verison as well? You can do that but the best pracise is to make these kind of breaking changes incrementally.

Reply:
we are using openapi generator plugin which is kind of not compatible with gradle 4.10, and even spring boot plugin did have some issues with gradle version.

Reply:
Ok, I see no harm in upgrading Gradle version.